mypyc/emitwrapper.py

```diff
@@ -26,18 +27,25 @@ def generate_wrapper_function(fn: FuncIR, emitter: Emitter) -> None:
         arg = real_args.pop(0)
         emitter.emit_line('PyObject *obj_{} = self;'.format(arg.name))
 
+    optional_args = []
+    for arg in fn.args:
+        if arg.optional:
+            optional_args.append(arg)
```

Comment:
Use a list comprehension to clean this up: `optional_args = [arg for arg in fn.args if arg.optional]`